packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamWizard.tsx

```diff
@@ -122,24 +121,23 @@ const ChallengeTeamWizard: React.FunctionComponent<
   const { data: challenge } = useGetEntityChallenge(projectId)
   const participantTeamId = challenge ? challenge.participantTeamId : EMPTY_ID
   const userId = userProfile ? userProfile.ownerId : EMPTY_ID
+
   // Verify that user is a member of the participant team
-  if (participantTeamId && userId) {
-    getMembershipStatus(participantTeamId, userId)
-      .then(challengeTeamMembershipStatus => {
-        if (!challengeTeamMembershipStatus.isMember && accessToken) {
-          addTeamMemberAsAuthenticatedUserOrAdmin(
-            participantTeamId,
-            userId,
-            accessToken,
-          ).catch(error => {
-            setErrorMessage(error.reason)
-          })
-        }
-      })
-      .catch(error => {
+  const { data: challengeTeamMembershipStatus } = useGetMembershipStatus(
+    participantTeamId,
+    userId,
+  )
+  useEffect(() => {
+    if (!challengeTeamMembershipStatus?.isMember && accessToken) {
```

Comment:
Will run if `challengeTeamMembershipStatus` is undefined, which will happen when it's not fetched

```suggestion
    if (challengeTeamMembershipStatus && !challengeTeamMembershipStatus?.isMember && accessToken) {
```